src/help/help_test.go

```diff
@@ -1,11 +1,23 @@
 package help
 
 import (
+	"os"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
+func TestMain(t *testing.M) {
+	f, err := os.Create(".plzconfig")
```

Comment:
You could just use `os.WriteFile` for this which is a couple of lines shorter ;-)